modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc

```diff
@@ -76,6 +77,30 @@ A clone of the Git repository is present in the filesystem of the new workspace.
 +
 The workspace has a unique URL: `pass:c,a,q[{prod-url}]#workspace__<unique_url>__`.
 
+.. Using a direct URL to `devfile`:
+[source,subs="+quotes,+attributes,+macros"]
++
+----
+pass:c,a,q[{prod-url}]#__<url_to_devfile>__
+----
++
+[TIP]
+====
+You can pass Personal Access Token to the URL to have access to private repositories:
```

Comment:
maybe we can add info that this would work for GitHub, GitLab, Bitbucket, Azure, and other types repositories that support this type of OAuth

Reply:
done